.gitignore

```diff
@@ -0,0 +1,4 @@
+.node_modules/
+test.html
+test.md
+node_modules
```

Comment:
- [ ] you are only required to ignore the node_modules folder and test.md file, kindly ensure that is what you are doing at the moment.
 - you can ignore both by adding these two lines of code below to your `.gitignore` file 
> - test.md
> - node_modules

Reply:
Thank you.
However, you are not really professional or technical here. As you can see, I ignored files I don't want on my GitHub and you are commenting on this. Is the .gitignore not meant for this reason? Try being reasonable next time. Thank you😎.

Reply:
hi @Paulinelali i understand that having multiple requests on a project you feel you have completed can be draining and stressful becasue i have been in your position severally, however with regards to this request the project requirement for reviewers states that you are to only ignore two files which was what i pointed out to you basically. yes you added the required lines but adding more lines which has no usefulness to the project doesn't make your work look clean or cut through which is what i only tried to point out to you with my review, i am sorry for any inconveniences.